packages/syft/src/syft/core/node/common/action/smpc_action_functions.py

```diff
@@ -0,0 +1,484 @@
+# stdlib
```

Comment:
Some of the maps that were kept in `smpc_action_message.py` file should be kept separated